doc/src/tutorial/howtosolve.rst

```diff
@@ -0,0 +1,34 @@
+===========================================
+ How to solve algebraic equations in Sympy
+===========================================
+
+
+Sympy solves algebraic equations with the help of the following functions
+
+* `solve()`
+
+* `solveset()`
+
+* `linsolve()`
+
+* `nonlinsolve()`
+
+* `solve\_decomposition()`
+
+* `solvify()`
```

Comment:
dsolve can also be mentioned.

It is worth mentioning how to solve linear systems with the matrix methods, as an alternative to solve() and linsolve().

Reply:
Yes, and there's also `rsolve`, `diophantine`, `solve_univariate_inequality`, `roots`, `Poly.nroots`, `RootOf`.

Reply:
Sure thing! Will add those too.

Reply:
What if I just leave a link to the [inequality solvers](https://docs.sympy.org/latest/modules/solvers/inequalities.html) page instead of restating their features here? I feel there's nothing to add and it'll be an overkill.